file-formats/uipg/type/zif_aff_uipg_v1.intf.abap

```diff
@@ -91,21 +91,45 @@ INTERFACE zif_aff_uipg_v1
       "! $required
       tile_key            TYPE ty_catalog_item_key,
 
-      "! <p class="shorttext">Tile Display Format</p>
-      "! Describes how the tile is rendered during run-time
-      tile_display_format TYPE ty_tile_display_format,
-
       "! <p class="shorttext">Target Mapping Key</p>
       "! Target mapping key
       target_mapping_key  TYPE ty_catalog_item_key,
 
     END OF ty_tile_assignment.
 
 
+  "! <p class="shorttext">UIAD ID</p>
+  "! Launchpad App Descriptor Item ID
+  TYPES ty_uiad_id      TYPE c LENGTH 32.
+
+
+  "! <p class="shorttext">UIAD Tile ID</p>
+  "! Launchpad App Descriptor Item Tile ID
+  TYPES ty_uiad_tile_id TYPE c LENGTH 50.
+
+
+  TYPES:
+    "! <p class="shorttext">UIAD Assignment</p>
+    "! Assignment of a launchpad app descriptor (UIAD) item
+    BEGIN OF ty_uiad_assignment,
+
+      "! <p class="shorttext">UIAD ID</p>
+      "! ID of a launchpad app descriptor (UIAD) item
+      "! $required
+      uiad_id             TYPE ty_uiad_id,
+
```

Comment:
I am wondering whether users understand the term UIAD which is a transport type. 

Would it be helpful for users/developers to see something like "launchpad app descriptor" instead of "UIAD"?

Reply:
I don't like the name Launchpad App Descriptor, but from a UI perspective this is more in line how we name things. I mean in the ADT the folder for UIADs is called FLP App Descriptor Items, so it makes sense to have the same name here to be consistent.
--> renamed